README.md

```diff
@@ -9,6 +9,6 @@ It should not contain any high-level functionality.
 
 ## How to launch manually
 
-```$ docker -H <Hostname>.local run --name duckiebot-interface -v /data:/data --privileged --network=host -dit --restart unless-stopped -e ROBOT_TYPE=<ROBOT_TYPE> duckietown/duckiebot-interface:daffy-arm32v7```
+```$ docker -H DUCKIEBOT-NAME run --name duckiebot-interface-night -v /data:/data --privileged --network=host -dit --restart unless-stopped -e ROBOT_TYPE=duckiebot duckietown/dt-duckiebot-interface:daffy-lighting-system-arm32v7```
```

Comment:
This will change the documentation of `dt-duckiebot-interface` to serve only one application. The Docker image is also changed, this documentation is intended for any Duckiebot, not just lighting system ones.